util/compiler.h

```diff
@@ -37,4 +37,11 @@
 #define ALWAYS_INLINE
 #endif
 
+/* Use to mark fall through on switch statements as desired. */
+#if __GNUC__ >= 7
```

Comment:
I think that clang supports it too.
https://clang.llvm.org/docs/AttributeReference.html